templates/update.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/python3
```

Comment:
@hmlnarik Why is this script needed? Maybe to automate the indentation to four spaces for all templates (yet before it's passed to other scripts)?

If that's the case could it be updated to take the file to operate at as CLI arg?

Reply:
Thanks @iankko , omission on my part, sorry!

Reply:
Ok, thanks! All clear now